src/layouts/default/PlayerOSD/PlayerFullscreen.vue

```diff
@@ -74,7 +74,25 @@
             class="fullscreen-track-info"
           >
             <!-- title -->
+            <!-- radio station name -->
             <h2
+              v-if="
+                store.curQueueItem?.media_item &&
+                store.curQueueItem.media_item?.media_type == MediaType.RADIO
+              "
+              style="cursor: pointer; width: fit-content; display: inline"
+              class="title line-clamp-1"
+              @click="
+                store.curQueueItem?.media_item
+                  ? radioNameClick(store.curQueueItem.media_item as Radio)
+                  : ''
+              "
+            >
+              {{store.curQueueItem.media_item.name}}
```

Comment:
```suggestion
              {{ store.curQueueItem.media_item.name }}
```

Reply:
Ah yeah, must remember to yarn lint!  That's fixed now.